js/drawPuck.js

```diff
@@ -24,10 +33,10 @@ function drawPuck() {
  *
  * @param {number} cellIndex
  */
-function onCellClick(cellIndex) {
+function onCellClick(cellIndex, additionalInfo) {
```

Comment:
This is actually not passed to the function, so it's always undefined. Let's remove it from this Pull Request for now, and let's work on this once this PR is finished.